### PR TITLE
{bio}[system/system] picard v2.22.1

### PR DESCRIPTION
**Pull request by bioteamMichael**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/picard/picard-2.22.1-Java-11.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Pavel Grochal (INUITS)
+##
+
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.22.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://sourceforge.net/projects/picard'
```
Comment (Micket) [inline easybuild/easyconfigs/p/picard/picard-2.22.1-Java-11.eb]: We should switch to https://broadinstitute.github.io/picard/  sourceforge page is dead.
Would you mind also switching the other 2 configs to use this new URL?

**Comment by lexming** [comment]
Re-triggering travis...

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node364.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/023a363550e584f513776032556d494b for a full test report.

**Comment by lexming** [comment]
Going in, thanks @bioteamMichael !

**Pull request merged**